src/components/NavBar.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { NavLink } from "react-router-dom";
+
+export default function NavBar() {
+  return (
+    <ul>
+      <li>
+        <NavLink to="/">Home </NavLink>
+      </li>
+      <li>
+        <NavLink to="/ahha">Archive </NavLink>
```

Comment:
please rename the link to

Reply:
Oh my god!! How did I not see this? UNACCEPTABLE

Reply:
Update it Thanks for the Input :)